### PR TITLE
Blog App: validations and model specs

### PR DESCRIPTION
**Pull request by itsmraga-hub**
# 💯 Achievements

- Use validations for models.
- Write unit tests for models.

### CODE REVIEW

File: spec/models/like_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+
+    @post = Post.create(author: @post, title: 'Hello', text: 'This is my first post')
+
+    @comment = Comment.create(post: @comment, author: @user, text: 'Hi Tom!')
+
+    @like = Like.create(author: @author, post: @post)
+  end
+
+  it 'likesCounter wi' do
```
Comment (rustam-tolipov) [inline spec/models/like_spec.rb]: [OPTIONAL] It's great you added a test for likesCounter. However, I couldn't catch the name of the test..., I would suggest adding a clear description to your test :).

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  # it 'Photo should be present' do
+  #   subject.photo = nil
+  #   expect(subject.photo).to_not be_valid
+  # end
+
+  # it 'Bio should be present' do
+  #   subject.bio = nil
+  #   expect(subject.bio).to_not be_valid
+  # end
```
Comment (rustam-tolipov) [inline spec/models/user_spec.rb]: - [x] I strongly recommend you to remove unused comments from your code. Kindly remove them if they are not necessary :)

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @itsmraga-hub,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @itsmraga-hub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**